src/app/onion/learning-object-builder/components/scaffold/add-child/add-child.component.ts

```diff
@@ -52,7 +52,8 @@ export class AddChildComponent implements OnInit, OnDestroy {
    */
   async getLearningObjects(filters?: any, query?: string): Promise<LearningObject[]> {
     this.loading = true;
-    const draftObjects = await this.learningObjectService.getDraftLearningObjects(this.child.author.username, { ...filters, text: query })
+    const draftObjects = await this.learningObjectService.getDraftLearningObjects(this.child.author.username || this.auth.username,
```

Comment:
I don't think we want to use the auth.username as the alternative because it looks like we are specifically getting draft objects for the learning object's children. Can we guarantee that `this.auth.username` is the owner of the child object? Because I though a lot of the implementations where this.auth is actually the user logged in.